docs/en/observability/cloud-monitoring/aws/monitor-amazon-ec2.asciidoc

```diff
@@ -0,0 +1,372 @@
+available with Microsoft Windows and Linux distributions like Ubuntu, Red Hat
+Enterprise Linux, CentOS, SUSE, Debian, and Amazon Linux. Amazon offers Amazon
+Elastic Block Store (EBS) volumes to EC2 instances for persistent block-level
+storage. EC2 can also be used with other AWS services like Auto Scaling.
+
+Amazon EC2 instances can be run in various locations. The location is composed
+of AWS Regions and Availability Zones (AZ). AZs are distinct locations that are
+insulated from failures to other AZs, but provide lower latency network
+connectivity. Each region is geographically dispersed and can consist of one or
+more AZs.
+
+Like most AWS services, Amazon EC2 sends its metrics to Amazon CloudWatch. The
+Elastic {integrations-docs}/aws/ec2[Amazon EC2 integration] collects metrics
+from Amazon CloudWatch using {agent}.
+
+CloudWatch, by default, uses basic monitoring that publishes metrics at
+five-minute intervals. You can enable detailed monitoring to increase that
+resolution to one-minute, at an additional cost. To learn how to enable detailed
+monitoring, refer to the
+https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-cloudwatch-new.html[Amazon EC2 documentation].
+
+CloudWatch does not expose metrics related to EC2 instance memory. You can
+install {agent} on the EC2 instances to collect detailed system metrics.
+
+[discrete]
+[[get-started-ec2]]
+== Get started
+
+To collect EC2 logs and metrics, you typically need to install the Elastic
+{integrations-docs}/aws/ec2[Amazon EC2 integration] and deploy an {agent}.
+
+Expand the **quick guide** to learn how, or skip to the next section if your
+data is already in {es}.
+
+//Sets the integration name used in the quick guide
+:integration-name: Amazon EC2
+
+include::{observability-docs-root}/docs/en/shared/integrations-quick-guide.asciidoc[]
+
+//Clears the integration name
+:integration-name!:
+
+{agent} is currently the preferred way to add EC2 logs and metrics. For other
+ways, refer to {cloud}/ec-cloud-ingest-data.html[Adding data to {es}].
+
+[discrete]
+[[dashboard-ec2]]
+== Dashboards
+
+{kibana-desc}
+For example, to see an overview of your EC2 instance metrics in {kib}, go to
+the **Dashboard** app and navigate to the **[Metrics AWS] EC2 Overview**
+dashboard.
+
+[role="screenshot"]
+image::images/ec2-overview-dashboard.png[Screenshot showing the EC2 overview dashboard]
+
+[discrete]
+== Metrics to watch
+
+Amazon EC2 provides a large selection of instance types. Monitoring key metrics
+about your instances is a cost-effective way to determine whether to upgrade,
+change, or downsize your instances.
+
+This section describes the key metrics that you should watch, organized by
+category. For a full list of exported fields, refer to the
+{integrations-docs}/aws/ec2[Amazon EC2 integration docs].
+
+[discrete]
+=== CPU utilization
+
+Here are the key CPU utilization metrics you should monitor.
+
+|===
```

Comment:
Remove this table and point to the fields reference in the integration's overview page instead.